tests/backend-configuration-test.js

```diff
@@ -192,7 +192,7 @@ describeApplication('With valid backend configuration', () => {
         });
 
         it('shows an error message', () => {
-          expect(SettingsPage.errorText).to.equal('an error has occurred');
+          expect(SettingsPage.toast.errorText).to.equal('an error has occurred');
```

Comment:
awh, does this mean there's no inline form errors?

Reply:
There are client-side validations to make sure the fields aren't empty, but yes, any server error just shows the toast.